### ISSUE TITLE
Email notifications could have a delay

### ISSUE THREAD

**Issue by schonarth** [RocketChat/Rocket.Chat#14309]
I just disabled the email notification feature since it happens quite often that a RocketChat new message notification pops up and right the next second I also get an email about it.

Important email started to drown in the middle of all these notifications -- and if I just wrote filters to put them away, they would serve no purpose, so better to not receive them in the first place. 

IMHO these email notifications could have a delay, like if the message goes unread for 1, 5, 10 or n minutes, THEN send out that email. That would greatly reduce the amount of emails sent, making them more relevant!

**Comment by softwaredoug** [comment]
I totally agree. Rocket chat generates a lot of email. I'm usually moments away from seeing the message In the app. Can there be a configurable lag before I get an email notification?

**Comment by dosmanak** [comment]
It seems this issue should be resolved by this PR https://github.com/RocketChat/Rocket.Chat/pull/17357

**Comment by Geza-Kiss-Shapr3d** [comment]
Also, when you make a comment on a PR, sometimes you realize something could be improved about it. It would be great if there was a fixed (e.g. 1 minute) delay before sending a notification about a comment after it was last modified. (Modifying it should prolong the delay.)

**Comment by wokawoka** [comment]
@dosmanak if I can ask after some months, what did you mean that the issue should be solved with  PR #17357? 
Is it possible to set rocketchat to wait for a small delay and combine different offline messages in one single e-mail notification?
I've extensively looked for such an option in the admin settings but I couldn´t find anything.

Thank you!

**Comment by dosmanak** [comment]
> @dosmanak if I can ask after some months, what did you mean that the issue should be solved with PR #17357?
To be honest,  I don't know.  The description seemed like it could resolve the notification issue in general so I found it good idea to link here. Perhaps it was a mistake.

**Comment by wokawoka** [comment]
@dosmanak alright, thank you anyway.
Do anybody else by chance know if a way to combine more offline messages into a single email message exist at the moment?
Thank you!
